evaluate.py

```diff
@@ -86,6 +86,7 @@ def play_match(black_model, white_model, games, sgf_dir):
             if active.is_done():
                 fname = "{:d}-{:s}-vs-{:s}-{:d}.sgf".format(int(time.time()),
                                                             white_name, black_name, i)
+                active.set_result(active.root.position.result(), was_resign=False)
```

Comment:
what was this for?

Reply:
the result string doesn't get set otherwise